modules/apps/frontend-js/frontend-js-test/src/testFunctional/paths/A11yTool.path

```diff
@@ -26,23 +27,13 @@
 	<td></td>
 </tr>
 <tr>
-	<td>VIOLATION_PANEL_ITEM_SEVERITY</td>
-	<td>//*[contains(@class,'a11y-panel')]//*[contains(@class,'list-group-subtext') and contains(.,'${key_text}')]</td>
-	<td></td>
-</tr>
-<tr>
-	<td>HIGHLIGHTED_ELEMENT</td>
-	<td>//*[contains(@class,'a11y-overlay')]//*[contains(@class,'a11y-backdrop')]</td>
-	<td></td>
-</tr>
-<tr>
-	<td>HIGHLIGHTED_ELEMENT_COUNT</td>
-	<td>//*[contains(@id, 'a11yContainer')]//*[contains(@class, 'a11y-overlay')][${key_index}]</td>
+	<td>VIOLATION_PANEL_RULE_LIST_INDEX</td>
+	<td>//*[contains(@class,'list-group-item')][${key_index}]//*[contains(@class,'rule-list-item') and contains(.,'${key_text}')]</td>
```

Comment:
actually, still debatable...I just realized you have a path named `VIOLATION_PANEL_ITEM_SEVERITY`. You can decide since you have more visibility on the a11y tool what would make more sense.

Reply:
hmmm I think impact would be good for consistency. Ill change it for the other paths as well